lib/dataset/string.cpp

```diff
@@ -115,9 +111,8 @@ template <class Key, class Value>
 std::string dict_to_string(const SizedDict<Key, Value> &view) {
   std::stringstream ss;
   ss << "<scipp.Dict>\n";
-  for (const auto &[key, item] : view) {
-    ss << "  " << key << ": " << to_string(item) << "\n";
-  }
+  for (const auto &[key, item] : view)
+    ss << "  " << key << ": " << format_variable(item) << "\n";
```

Comment:
Are you sure this function is not used for datasets?

Reply:
nice catch. not sure why that changed, I think it's left over from a previous solution, but I am not using thing anymore, so that should just be untouched.